content/pt/docs/concepts/instrumentation/libraries.md

```diff
@@ -0,0 +1,465 @@
+o uso da sua biblioteca e pense em uma maneira apropriada de modelar esses
+dados. Algumas considerações incluem:
+
+- Trechos e hierarquias de trecho
+- Atributos numéricos em rastros (como alternativa a métricas agregadas)
+- Eventos em rastros
+- Métricas agregadas
+
+Por exemplo, se sua biblioteca está fazendo requisições a um banco de dados,
+crie rastros apenas para a requisição lógica ao banco de dados. As requisições
+físicas pela rede devem ser instrumentadas nas bibliotecas que implementam essa
+funcionalidade. Além disso, é preferível capturar outras atividades, como a
+serialização de objetos/dados, como eventos em rastros, ao invés de rastros
+adicionais.
+
+Siga as convenções semânticas ao definir atributos em rastros.
+
+## Quando **não** instrumentar
+
+Algumas bibliotecas atuam como camadas finas que encapsulam chamadas de rede. Há
+uma grande chance de que o OpenTelemetry já tenha uma biblioteca de
+instrumentação para o cliente RPC subjacente (confira o
+[registry](/ecosystem/registry/)). Nesse caso, pode não ser necessário
+instrumentar a biblioteca que encapsula essas chamadas. Como diretriz geral, só
+instrumente sua biblioteca em seu próprio nível.
+
+Não instrumente se:
+
+- sua biblioteca é um proxy simples em cima de APIs documentadas ou
+  autoexplicativas
+- _e_ o OpenTelemetry já tem instrumentação para as chamadas de rede subjacentes
+- _e_ não existem convenções que sua biblioteca deva seguir para enriquecer a
+  telemetria
+
+Se estiver em dúvida - não instrumente - você sempre pode fazê-lo mais tarde,
+quando perceber que é necessidade.
+
+Se optar por não instrumentar, ainda pode ser útil fornecer uma maneira de
+configurar _handlers_ do OpenTelemetry para a instância interna do cliente RPC.
+Isso é essencial em linguagens que não suportam instrumentação totalmente
+automática e ainda é útil em outras.
+
+O restante deste documento fornece orientações sobre o que e como instrumentar,
+caso decida fazê-lo.
+
+## OpenTelemetry API
+
+O primeiro passo é adicionar a dependência do pacote OpenTelemetry API.
+
+O OpenTelemetry possui [dois módulos principais](/docs/specs/otel/overview/) -
+API e SDK. A API do OpenTelemetry é um conjunto de abstrações e implementações
+não operacionais. A menos que sua aplicação importe o SDK do OpenTelemetry, sua
+instrumentação não faz nada e não impacta o desempenho da aplicação.
+
+**Bibliotecas devem usar apenas a API do OpenTelemetry.**
+
+Você pode estar com receio de adicionar novas dependências, então aqui estão
+algumas considerações para ajudar a minimizar problemas com dependências:
+
+- A API de rastros do OpenTelemetry alcançou estabilidade no início de 2021,
+  seguindo a
+  [Convenção semântica 2.0](/docs/specs/otel/versioning-and-stability/), e
+  levamos a estabilidade da API a sério.
+- Ao definir dependências, use a versão estável da API do OpenTelemetry (1.0.\*)
+  e evite atualizá-la, a menos que precise de novos recursos.
```

Comment:
```suggestion
  e evite atualizá-la, a menos que precise usar novas funcionalidades.
```

Usar `recursos` aqui está certo, mas como há menções de `recursos` the otel que querem dizer algo bem diferente, talvez seja melhor usar funcionalidade aqui pra não causar nenhuma confusão